config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
+# information see https://developers.google.com/web/updates/2018/06/feature-policy
+#
+# Rails.application.config.permissions_policy do |f|
+#   f.camera      :none
+#   f.gyroscope   :none
+#   f.microphone  :none
+#   f.usb         :none
+#   f.fullscreen  :self
+#   f.payment     :self, "https://secure.example.com"
+# end
```

Comment:
The entire file is commented, do we need to commit it?

Reply:
🤷 placeholder file that came out of `rails app:update` - i'd be inclined to keep it! what do you think?